### ISSUE TITLE
Should we include rails-erd?

### ISSUE THREAD

**Issue by lexi-lambda** [philosophie/suspenders#83]

**Comment by taboularasa** [comment]
I vote :-1: I like rails erd but we don't need it all the time and it's broken for people who don't have graphviz setup

**Comment by jdwolk** [comment]
I vote :thumbsup: since it's useful on virtually every project -- what cases have you not used it, @taboularasa?

**Comment by taboularasa** [comment]
@jdwolk I've not used it at all in a while, when I have used it the majority of the team could not run the rake task because their graphviz was broken so I ended up slacking the pdf when someone else needed it. maybe you're biased just coming off spree project?

**Comment by jdwolk** [comment]
@taboularasa ordinarily I might agree with you, but @wykhuh just used on our most recent project without any prompting. Also I can't think of a project in recent memory where it hasn't been useful

**Comment by wykhuh** [comment]
@taboularasa @jdwolk I used rails-erd on my two recent projects, CSL and staying sharp. Cody and I installed it on CSL when our data model was becoming very complicated and we wanted to see if there was a way to simplify it. 

I wished I knew about rails-erd during my first project AiW. Jumping into an existing project, it was hard to get an overview of the entire project. An erd diagram would have been helpful.

rails-erd mentioned that people should install  Graphviz  2.26+.
